.credo.exs

```diff
@@ -165,7 +165,12 @@
          files: %{
            excluded: [
              "lib/elixir_internal_certificate/account/accounts.ex",
+             "lib/elixir_internal_certificate/scraper/schemas/search_result.ex",
+             "lib/elixir_internal_certificate/scraper/scrapers.ex",
+             "lib/elixir_internal_certificate_worker/scraper/user_search_worker.ex",
```

Comment:
I'm not sure something wrong but it seems we keep extend this list when we adding new modules, should we invest to find the root problem in the codebase @nvminhtue

Reply:
@andyduong1920 sounds good, I will do

Reply:
Leaving for now, still have not figured the solution out yet. I will update you whenever find out the way to fix it entirely.

Reply:
Noted @nvminhtue